src/SilverStripe/BlowGun/Credentials/BlowGunCredentials.php

```diff
@@ -2,9 +2,10 @@
 
 namespace SilverStripe\BlowGun\Credentials;
 
-use Aws\Common\Credentials\Credentials;
+use Aws\Credentials\CredentialProvider;
+use RuntimeException;
 
-class BlowGunCredentials extends Credentials
+class BlowGunCredentials
```

Comment:
This hardly (and probably never was) used as a `Credential`. A quick glance seems to indicate that all the `self::` references still pointing to this class and the `CredentialProvider`. Should we rename the class to something that wouldn't cause confusion in the future?

Reply:
yeah I think it's more of a `Util` class than anything, will change it to be less confusing

Reply:
fixed!